ephios/core/templates/core/event_list.html

```diff
@@ -80,24 +80,8 @@ <h1>{% translate "Events" %}
         </form>
         {% if mode == "calendar" or mode == "day" %}
             {% if perms.core.add_event %}
-                <div class="dropdown">
-                    <button class="btn btn-primary btn-sm dropdown-toggle d-none d-md-inline-block" type="button"
-                            id="eventCreateButton"
-                            data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        <span class="fa fa-plus"></span>
-                        {% translate "Add event" %}
-                    </button>
-                    <div class="dropdown-menu dropdown-menu-right" aria-labelledby="eventCreateButton">
-                        {% for eventtype in eventtypes %}
-                            <a class="dropdown-item" href="{% url "core:event_create" eventtype.pk %}">
-                                {% blocktranslate trimmed with title=eventtype.title %}
-                                    Add {{ title }}
-                                {% endblocktranslate %}</a>
-                        {% endfor %}
-                    </div>
-                </div>
+                {% include "core/fragments/event_list_add_event_button.html" %}
```

Comment:
what was the reason again that we include the button here for calendar and day mode and below for the list view?

Reply:
kk, then lgtm